### ISSUE TITLE
Cannot download videos without xattributes enabled

### ISSUE THREAD

**Issue by taratatach** [Unrud/video-downloader#48]
Downloading a video on my ext4 drive has been failing all the time for a while with the following error:
```
ERROR: ERROR: This filesystem doesn't support extended attributes. (You may have to enable them in your /etc/fstab)
```

It used to work fine and I think xattributes shouldn't be a requirement to download videos.

**Comment by Unrud** [comment]
Fixed in version 0.5.8

**Comment by taratatach** [comment]
Awesome, thank you! :)
